WooCommerce/Classes/POS/Utils/PreviewHelpers.swift

```diff
@@ -54,21 +54,20 @@ final class PointOfSalePreviewItemService: PointOfSaleItemServiceProtocol {
 }
 
 final class PointOfSalePreviewItemsController: PointOfSaleItemsControllerProtocol {
-    @Published var itemListState: ItemListState = .initialLoading
-    var itemListStatePublisher: any Publisher<ItemListState, Never> { $itemListState }
-
-    var allItems: [POSItem] = []
+    @Published var itemsViewState: ItemsViewState = ItemsViewState(containerState: .loading,
+                                                                   itemsStack: ItemsStackState(root: .loading([])))
+    var itemsViewStatePublisher: any Publisher<ItemsViewState, Never> { $itemsViewState }
 
     func loadInitialItems() async {
-        itemListState = .loaded(mockItems)
+        itemsViewState = ItemsViewState(containerState: .content, itemsStack: ItemsStackState(root: .loaded(mockItems)))
     }
 
     func loadNextItems() async {
-        itemListState = .loading(mockItems)
+        itemsViewState = ItemsViewState(containerState: .content, itemsStack: ItemsStackState(root: .loading(mockItems)))
     }
 
     func reload() async {
-        itemListState = .loaded([])
+        itemsViewState = ItemsViewState(containerState: .content, itemsStack: ItemsStackState(root: .loading([])))
```

Comment:
nit: if we want to match the previous preview behavior:
```suggestion
        itemsViewState = ItemsViewState(containerState: .content, itemsStack: ItemsStackState(root: .loaded([])))
```

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/14727/commits/2a94b520944a91a5db51ba2ad6c711aedc9a6de3.